pkg/api/v1alpha1/cluster_types.go

```diff
@@ -41,6 +41,9 @@ const (
 	// cluster object.
 	managementAnnotation = "anywhere.eks.amazonaws.com/managed-by"
 
+	// ManagementComponentsVersionAnnotation is an annotation applied to an EKS-A management cluster pointing to the current version of EKS management components.
```

Comment:
This was intended to store the version of the management components referred to in the `EKS-A bundle`, and using that to search for the correct `bundles` object from the cluster.

I see that it would make more sense to use the `EKSARelease` object name used for the management components here though.

Reply:
Some clarification, did you mean the EKSARelease object name or the semantic version [here](https://github.com/aws/eks-anywhere/pull/7250#discussion_r1444635572) object name?

Reply:
Discussed offline, we'll go with the version.